base/init.jl

```diff
@@ -0,0 +1,49 @@
+# We need to know the Pkg.jl URI and version for bootstrapping
+const PKG_URI = "https://github.com/rofinn/Pkg.jl"
+const PKG_VERSION = "v0.2.0"
+const VERS = "v$(VERSION.major).$(VERSION.minor)"
+
+_pkg_home() = abspath(JULIA_HOME, "..", "share", "julia", "site", VERS, "Pkg.jl")
+_pkg_require() = abspath(JULIA_HOME, "..", "etc", "julia", "REQUIRE")
+
+"""
+    initialize(interactive)
+
+Initializes a collection of core julia packages.
+If `interactive=true` you will be prompted to select the packages to include by
+editing the default REQUIRE file included in base julia.
+"""
+macro initialize(arg)
```

Comment:
Yeah, I made this a macro cause I was initially trying to do `using Pkg`, but I think making it a function and keeping the `Pkg = include(...)` is the better choice.